modules/ROOT/pages/depl-examples/ubuntu-compose/ubuntu-compose-eval.adoc

```diff
@@ -0,0 +1,529 @@
+
+* Basic knowledge about terms like: +
+server, hostname, localhost, DNS, DHCP, network ect. +
+If you are not sure, just use a search engine for more details.
+
+* Command line (bash) basics like: +
+ssh, actions on files, edit files via nano or vi etc.
+
+* Search via a search engine like google for OS management related topics like: +
+Setting or changing hostnames, IP configuration, installing SW packages etc.
+
+* Heard about docker and docker compose: +
+You are interested in using it, possibly for the first time.
+
+=== Software Stack
+
+For the OS, *Ubuntu LTS 24.04* has been selected, but it will also work with Ubuntu LTS 22.04. On purpose, we reference to https://ubuntu.com/download/desktop[Ubuntu Desktop] as that has a GUI including a browser preinstalled. The office suite that is offered to be installed with Ubuntu Desktop is not a prerequisite and can be omitted during the OS installation process if a fresh setup is planned.
+
+If you already have a computer running Ubuntu Desktop LTS 24.04, you can use that one as long it meets the requirements listed below. 
+
+Note that this guide expects:
+
+* The prerequisite of a computer with the installed OS is met and any required software other than Infinite Scale is installed and preconfigured and all the software is updated to the latest version.
+
+* You have at minimum shell access from within the GUI and you can reach your host via ssh from another computer in your network. `sudo` permissions are required in some steps later in the setup.
+
+== Limitations
+
+These are limitations that are defined by the setup chosen.
+
+[#accessing-ocis]
+Accessing Infinite Scale::
+--
+. The setup requires in the first step a system that has a GUI. Though you can expand the setup as described in xref:access-from-the-local-network[Access From the Local Network], a headless system (no GUI) like it is used in an production environment is covered in a follow up documentation.
+. Based on the goal of setting up an Infinite Scale instance as fast as possible, any access to the instance is in the first step *only* possible from the computer Infinite Scale is installed on and in the second step from the internal network ([.lime]#green#) the server is connected to.
++
+Setting up access from the outside aka internet to the server using an external resolvable domain like `ocis.mydomain.com` and routing networks ([.red]#red#) is *not* part of this documentation but can be configured.
++
+image::depl-examples/ubuntu-compose/ubuntu-test-install.drawio.svg[Network Overview, width=400]
+
+. To access Infinite Scale from your computer or from your local network, the domain name `ocis.owncloud.test` and other required subdomains are used which are not resolvable from external networks.
+
+. Because no validated certificates are used for this environment, any browser accessing Infinite Scale will get invalid certificate warnings that must be accepted upfront, which is safe to do so.
```

Comment:
```suggestion
. Because no validated certificates are used for this environment, any browser accessing Infinite Scale will get invalid certificate warnings that must be accepted upfront, which is safe to do.
```